src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -287,8 +287,8 @@ private void waitForLoaded()
         WebDriverWrapper.waitFor(()-> {
```

Comment:
This change acknowledges the use case for assay run updates;  when uploading assay run data, no entityTypeSelect is present so there needs to be a condition here that doesn't require an entityTypeSelect to be shown.
Likewise, the grid isn't shown unless the entityTypeSelect has a value/the component has a target to get the target table schema from- so isGridVisible() will always mean the entityTypeSelect has a value.
If the user arrives at a page without the URL param to populate entityTypeSelect, it will be empty (this case happens in the core-components view, and can happen in SM and Biologics.